extensions/inline/sub.rng

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+
+    <sch:ns prefix="libero" uri="http://libero.pub"/>
+
+    <define name="libero.inline.full.highlighting" combine="choice">
+        <ref name="libero.sub.full"/>
+    </define>
+
+    <define name="libero.inline.limited.highlighting" combine="choice">
+        <ref name="libero.sub.limited"/>
+    </define>
+
+    <define name="libero.sub.full">
+
+        <element name="sub">
+            <ref name="libero.sub.full.attributes"/>
+            <ref name="libero.sub.full.content"/>
+        </element>
+
+    </define>
+
+    <define name="libero.sub.limited">
+
+        <element name="sub">
+            <ref name="libero.sub.limited.attributes"/>
+            <ref name="libero.sub.limited.content"/>
+        </element>
+
+    </define>
+
+    <define name="libero.sub.full.attributes">
+        <empty/>
+    </define>
+
+    <define name="libero.sub.full.content">
+        <ref name="libero.inline.full"/>
+    </define>
+
+    <define name="libero.sub.limited.attributes">
+        <empty/>
+    </define>
+
+    <define name="libero.sub.limited.content">
+        <ref name="libero.inline.limited"/>
+    </define>
+
+    <sch:pattern>
+        <sch:rule context="libero:sub">
+            <sch:assert test="not(ancestor::libero:sub)">
```

Comment:
Probably want to not allow it inside `sup` too (and the reverse). We can merge this one first though and add it in the other.

Reply:
@thewilkybarkid is there a reason why we would nest any elements with `sub` or `sup`?

Reply:
I meant you probably shouldn't have subscript inside superscript...

Reply:
Misunderstood this, question is like https://github.com/libero/schemas/pull/18#issuecomment-464003022.

Not aware of any scenarios where you would, so safe to not allow it. The extensibility model allows you to add it in if needed (so `libero.sub.full.content` would need to contain `<empty/>`).